activation/validation.go

```diff
@@ -349,6 +362,7 @@ type verifyChainOpts struct {
 	assumedValidTime time.Time
 	trustedNodeID    types.NodeID
 	logger           *zap.Logger
+	prioritizedCall  bool
```

Comment:
I think this doesn't need to be an option here. The `verifyChain` method should always be prioritized.

Reply:
But `verifyChain` is called inside `findFullyValidHighTickAtx`, which is called in 2 places. Second is [here](https://github.com/spacemeshos/go-spacemesh/blob/develop/activation/post.go#L408) So do we want to do it prioritized too or I missed smth?

Reply:
We want to prioritize whenever the code is doing something for itself. It needs to be more selfish and less altruistic to have bigger chances of doing its part of the protocols on time (like registering in poet on time etc.). The other use is also doing something for itself (searching for a commitment ATX) so it's fine to prioritize there too.